content/docs/configuration.md

```diff
@@ -256,31 +256,36 @@ The acceptable names are the same as for the package config:
  Key name                     | Type            | Description
 ------------------------------|-----------------|----------------------------------------------------------------------
  `debug`                      | bool            | enable debug logs
- `dry_run`                    | bool            | Do not perform any remote changes (pull requests or comments)
+ `dry_run`                    | bool            | do not perform any remote changes (pull requests or comments)
```

Comment:
I am not sure it works... https://github.com/packit-service/packit/issues/232

Reply:
removed